heartbeat/beater/heartbeat.go

```diff
@@ -89,6 +91,19 @@ func (bt *Heartbeat) Run(b *beat.Beat) error {
 	groups, _ := syscall.Getgroups()
 	logp.Info("Effective user/group ids: %d/%d, with groups: %v", syscall.Geteuid(), syscall.Getegid(), groups)
 
+	for _, m := range bt.config.Monitors {
+		target := make(map[string]interface{})
+		m.Unpack(target)
```

Comment:
we should create a small util for this, this is such a common use case .

Reply:
There's not really a great way to do this since go doesn't support generics (yet). If you have a proposal lmk though